featurebyte/worker/task/batch_feature_table.py

```diff
@@ -61,9 +68,25 @@ async def execute(self, payload: BatchFeatureTableTaskPayload) -> Any:
         )
         db_session = await self.session_manager_service.get_feature_store_session(feature_store)
 
-        batch_request_table_model = await self.batch_request_table_service.get_document(
-            payload.batch_request_table_id
-        )
+        if payload.batch_request_table_id:
+            batch_request_table_model = await self.batch_request_table_service.get_document(
+                payload.batch_request_table_id
+            )
+        else:
+            # Create a temporary batch request table
+            batch_request_table_payload = BatchRequestTableTaskPayload(
+                name="temporary_batch_request_table",
```

Comment:
Should this temporary table name include a unique identifier like `ObjectId` to avoid name collision?

Reply:
Oh I missed that there is now a `create_document` flag which will be set to False so this is not an issue.